src/item.cpp

```diff
@@ -4802,6 +4810,11 @@ std::string item::damage_symbol() const
     }
 }
 
+std::string item::dirt_symbol() const
```

Comment:
As this is only used in ` item::tname`, you don't need to expose it at all.

Reply:
I'm still not sure how to do this. I searched for the thing you mentioned but I don't see any relevant examples.

Reply:
think i got it